src/simulation.hpp

```diff
@@ -2595,22 +2656,40 @@ template <typename problem_t> void AMRSimulation<problem_t>::WritePlotFile()
 #else
 	int included_ghosts = std::min(nghost_cc_, nghost_fc_);
 #endif
-	amrex::Vector<amrex::MultiFab> mf = PlotFileMF(included_ghosts);
-	amrex::Vector<const amrex::MultiFab *> mf_ptr = amrex::GetVecOfConstPtrs(mf);
+	amrex::Vector<amrex::MultiFab> mf_cc = PlotFileMF_cc(included_ghosts); // create fc version, triggered in next line
+	// Adding plotfileMF fc
+	if constexpr (Physics_Indices<problem_t>::nvarTotal_fc > 0) {
+	}
+	//
+	amrex::Vector<const amrex::MultiFab *> mf_ptr = amrex::GetVecOfConstPtrs(mf_cc);
```

Comment:
This is an exception to my comments above. This variable name *should* be changed to mf_cc_ptr, since there is a mf_fc_ptr in the same function. So the name change is needed to distinguish the meaning of the two pointers.